src/components/Autocomplete/components/ComboBox/ComboBox.tsx

```diff
@@ -253,17 +253,20 @@ export default class ComboBox extends React.PureComponent<Props, State> {
       options.length === 0 &&
       emptyState && <div className={styles.EmptyState}>{emptyState}</div>;
 
+    const {disabled} = textField.props;
+    const active = !disabled && this.state.popoverActive;
+
     return (
       <div
         onClick={this.handleClick}
         role="combobox"
-        aria-expanded={this.state.popoverActive}
+        aria-expanded={active}
         aria-owns={this.state.comboBoxId}
         aria-controls={this.state.comboBoxId}
         aria-haspopup
         onFocus={this.handleFocus}
         onBlur={this.handleBlur}
-        tabIndex={0}
+        tabIndex={disabled ? -1 : 0}
```

Comment:
I just noticed that the Focus Rectangle is still visible with this tabIndex when navigating with keyboard.

I just tried quickly removing the tabIndex all-together and it looked like it worked because of the events bubbling from the text field. That was a quick test so we need to ensure it does work properly.

Also you might not need: 

```
.ComboBox > div:focus {
  outline: none;
}
```

If we do this. (This doesn't show up in the PR's changed files for some reason?) If we do need it however we should probably change to a classname so we don't rely on the div. Although like I said it might not be needed with the removal of the tab index.

Reply:
@dleroux If the attributes are moved, we'd have to fall back to the [ARIA 1.0 combobox pattern](https://www.w3.org/TR/wai-aria-practices/#combobox) instead of the current 1.1 model, since how the attributes are nested and related as parents or siblings changes the semantic structure of the whole widget. Here's some more detail on the current accessibility state: https://github.com/Shopify/polaris-react-deprecated/issues/1969